TemporaryProjects.14.0/StartPageExtenderPackage.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Threading;
+using System.Runtime.InteropServices;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Shell;
+using Task = System.Threading.Tasks.Task;
+
+namespace TemporaryProjects
+{
+    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
+    [InstalledProductRegistration("#110", "#112", "1.0", IconResourceID = 400)] // Info on this package for Help/About
```

Comment:
Do we really need this attribute on both packages? I think it causes duplication in Help -> About:
![image](https://user-images.githubusercontent.com/11444821/43406309-3b2b9d98-941c-11e8-9a66-a844146ce936.png)

Reply:
➡️ Fixed in 62ecd1abc1f4fa6ef81ed094f7e4180ea3933c1b